derive_builder_core/src/builder_field.rs

```diff
@@ -33,12 +33,14 @@ pub struct BuilderField<'a> {
     ///
     /// The corresonding builder field will be `Option<field_type>`.
     pub field_type: &'a syn::Ty,
-    /// Wether the builder implements a setter for this field.
+    /// Whether the builder implements a setter for this field.
     ///
     /// Note: We will fallback to `PhantomData` if the setter is disabled
     ///       to hack around issues with unused generic type parameters - at least for now.
     pub setter_enabled: bool,
     /// Visibility of this builder field, e.g. `syn::Visibility::Public`.
+    pub field_visibility: &'a syn::Visibility,
+    /// Visibility of this field's setter, e.g. `syn::Visibility::Public`.
     pub setter_visibility: &'a syn::Visibility,
```

Comment:
`BuilderField::setter_visibility` can be removed. It's only usage is one initialisation in the `default_builder_field!` macro.